runtime-modules/storage/src/lib.rs

```diff
@@ -14,7 +14,19 @@ pub type StorageWorkingGroupInstance = working_group::Instance2;
 pub(crate) type StorageWorkingGroup<T> = working_group::Module<T, StorageWorkingGroupInstance>;
 
 // Alias for the member id.
-pub(crate) type MemberId<T> = <T as membership::Trait>::MemberId;
+pub(crate) type MemberId<T> = <T as common::Trait>::MemberId;
+
+// Alias for the content id.
+pub(crate) type ContentId<T> = <T as common::Trait>::ContentId;
+
+/// Channel id representation
+pub(crate) type ChannelId<T> = <T as common::Trait>::ChannelId;
```

Comment:
This is not the correct place actually where this type should be defined. It will be available in the runtime primitives configuration and should be passed to the storage Trait as a configurable type.

Reply:
see here: https://github.com/Joystream/joystream/pull/2123/files#diff-00cafb309cb91b1faac60e76701a3ffc8a5c26aee603b17d81514383615a0c3fR45

Reply:
I made them configurable through the common trait for the purpose of reusability between new content directory and storage.